diff --git a/functions/fish_prompt.fish b/functions/fish_prompt.fish
index 994513d..d57037d 100644
--- a/functions/fish_prompt.fish
+++ b/functions/fish_prompt.fish
@@ -164,7 +164,5 @@ if test "$tide_prompt_transient_enabled" = true
     end
 
     bind \r _tide_enter_transient
-    bind \n _tide_enter_transient
     bind -M insert \r _tide_enter_transient
-    bind -M insert \n _tide_enter_transient
 end
